src/schema/paddle.ts

```diff
@@ -429,6 +465,34 @@ export const resolvers: IResolvers<unknown, AuthContext> = traceResolvers<
 
       return consolidated.filter(Boolean) as BasePricingPreview[];
     },
+    pricingPreviewByIds: async (
+      _,
+      { ids, locale }: PaddlePricingPreviewByIdsArgs,
+      ctx,
+    ): Promise<BasePricingWithoutMetadata[]> => {
+      if (!ids?.length) {
+        throw new ValidationError('No ids provided');
+      }
+
+      const preview = await getPlusPricePreview(ctx, ids);
+      const pricing = preview.details.lineItems.map((item) => ({
+        priceId: item.price.id,
+        price: getProductPrice(item, locale),
+        duration: getPricingDuration(item),
+        trialPeriod: item.price.trialPeriod,
+        currency: {
+          code: preview.currencyCode,
+          symbol: removeNumbers(item.formattedTotals.total),
+        },
+      }));
+
+      if (!pricing?.length) {
+        logger.error(`Pricing preview by ids ${ids} returned no items`); // to send an alert
```

Comment:
Please use `{ err }, message` syntax, and tag with some property which we can then add to critical alert:

```suggestion
        const error = new ValidationError('pricing returned no items')
        logger.error({ err: error, ids  }, error.message); 
```

Reply:
and then throw error

Reply:
```suggestion
        const err = new ValidationError('pricing returned no items')
        logger.error({ err, ids  }, error.message); 
```